rdrf/rdrf/templatetags/is_radio.py

```diff
@@ -0,0 +1,15 @@
+from django import template
+from django.forms.widgets import RadioSelect
+
+register = template.Library()
+
+
+@register.filter(name='is_radio')
+def is_radio(element):
+    if hasattr(element, "field"):
+        field = element.field
```

Comment:
Shouldn't we shortcut the function (return False) if there is no field on element?

Reply:
Updated.